<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsHotkeyComponent</class>
 <widget class="QFrame" name="CSettingsHotkeyComponent">
  <property name="windowTitle">
   <string>Hotkeys</string>
  </property>
  <layout class="QVBoxLayout" name="vl_SettingsHotkeys">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QTableView" name="tv_Hotkeys">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wi_Buttons" native="true">
     <layout class="QHBoxLayout" name="hl_Buttons">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="tb_ReloadHotkey">
        <property name="toolTip">
         <string>reload hotkey settings</string>
        </property>
        <property name="statusTip">
         <string>reload settings</string>
        </property>
        <property name="text">
         <string>reload</string>
        </property>
        <property name="icon">
         <iconset resource="../../misc/misc.qrc">
          <normaloff>:/pastel/icons/pastel/16/arrow-refresh.png</normaloff>:/pastel/icons/pastel/16/arrow-refresh.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hs_Buttons">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="pb_EditHotkey">
        <property name="text">
         <string>edit</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="pb_AddHotkey">
        <property name="text">
         <string>add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_RemoveHotkey">
        <property name="text">
         <string>remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tv_Hotkeys</tabstop>
 </tabstops>
 <resources>
  <include location="../../misc/misc.qrc"/>
 </resources>
 <connections/>
</ui>
